docs/getting-started/downloading-export-batches-keys.md

```diff
@@ -11,6 +11,84 @@ server. Keys can be grouped by region and the frequency of batching is
 configurable. We recommend batches are configured to be generated at most every
 4 hours, with a likely time being somewhere between 4 and 24 hours.
 
+# Consuming Export Batches
+
+## Index File Structure
+
+The index.txt file indicates which export files are available for a mobile device
+to consume. The index file contains relative path names for available archives.
+A client is typically given two configuration items for pointing at a server
+
+* The absolute location of the index.txt file
```

Comment:
Should we make a note that app developers are encouraged to use runtime configuration and **NOT** hardcode these values?